src/sgame/botlib/bot_nav.cpp

```diff
@@ -386,12 +386,26 @@ bool G_BotNavTrace( int botClientNum, botTrace_t *trace, const glm::vec3& start_
 	return true;
 }
 
-void G_BotAddObstacle( const glm::vec3 &mins, const glm::vec3 &maxs, qhandle_t *obstacleHandle )
+struct bbox_t {
+	glm::vec3 mins;
+	glm::vec3 maxs;
+};
+static std::map<int, bbox_t> savedObstacles;
```

Comment:
Can these be unordered map? I don't see any reason they need to be sorted. If so, please leave a comment on why

Reply:
Performance-wise it shouldn't matter because there are like 10 obstacles in a normal game map and absolute worst case there would be 958. I guess this brings a bit more determinism or something?